cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
-
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  it('should display the total number of pages when new rows are added', () => {
+    cy.createWorkers(4, 11);
```

Comment:
```suggestion
    cy.createWorkers(11);
```
modify this function to accepts only 1 argument, because now it is not clear what does it mean 4,11. Also, no need to add 11 users, it takes to much time, you can add 3 users and change table rows count to 5

Reply:
nothing changed